### ISSUE TITLE
GetRandomPostAsync returns the same post every time

### ISSUE THREAD

**Issue by Rasmus715** [Xwilarg/BooruSharp#48]
When upgrading from 3.4.0 NuGet package to 3.5.4 I've faced a problem where GetRandomPostAsync returns the same post depending on time you're calling the method. 
Code to reproduce:
```
using BooruSharp.Booru;

var booru = new Konachan();

for (var i = 0; i < 100; i++)
{
    var post = await booru.GetRandomPostAsync();
    Console.WriteLine($"Post ID: {post.ID}");
}
```
Stacktrace:
```
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Post ID: 347773
Unhandled exception. System.Net.Http.HttpRequestException: Response status code does not indicate success: 503 (Service Unavailable).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at BooruSharp.Booru.ABooru.GetJsonAsync(String url)
   at BooruSharp.Booru.ABooru.GetSearchResultFromUrlAsync(String url)
   at BooruSharp.Booru.ABooru.GetRandomPostAsync(String[] tagsArg)
   at Program.<Main>$(String[] args) in /Volumes/Media/Projects/ConsoleApp2/ConsoleApp2/Program.cs:line 7
   at Program.<Main>(String[] args)

Process finished with exit code 134.
```

Building the app using .NET SDK 6.0.202

**Comment by Xwilarg** [comment]
Thanks for the report, I was indeed able to reproduce the issue, I'll investigate that and come back to you

**Comment by Xwilarg** [comment]
This should be fixed with [version 3.5.5](https://www.nuget.org/packages/BooruSharp/3.5.5), you might have to wait a few minutes for the package to be indexed

**Comment by Rasmus715** [comment]
For me the issue is fixed. Thank you for operative response, @Xwilarg. Have a nice day :)
